NoiseStudies/Macros/The Average signal in the pos EM 1 (TS2 .cpp

```diff
@@ -0,0 +1,324 @@
+The Average signal in the pos EM 1 (TS2 / TS3 over 2) for bin number 1 is 415.36
```

Comment:
Why is this in a code file? Seems more like text?

Reply:
It is text, sorry. For some of the analysis I have my code cout different information that I need and I just save it to the Macro folder so that I have it. I removed it, but I have to push that through.